.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -25,6 +25,27 @@ Then
 - [ ] Checklist item
 - [ ] Meets the DOD
 
+**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)
+- [ ] User story is included
+- [ ] User role and type are identified
+- [ ] Acceptance criteria are included
+- [ ] Wireframes are included (if required)
+- [ ] Does this User Story need stakeholder approval?
+- [ ] Design / Solution is accepted by Product Owner
+- [ ] Dependencies are identified (technical, business, regulatory/policy)
+- [ ] Story has been estimated (under 13 spts)
+
+·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)
+- [ ] Passes developer unit and automated tests
+- [ ] UI meets accessibility requirements
+- [ ] Configuration changes are documented, documentation and designs are updated
+- [ ] Feature flagged if required
+- [ ] Passes code peer-review
+- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
+- [ ] Ticket is ready to be merged to main branch
```

Comment:
What is the difference between this and the "Approved by PO or Dev" pipeline in Zenhub?

Reply:
Will remove "Feature flagged if required".
These points were taken from Sprint Retro DOR/DOD session on Apr1. If we want to rewrite or remove it, we can do that.
I see the point that this is similar to the pipeline. Will it be better to have those check points there to tick ?